Make/MkFlags.ini

```diff
@@ -14,6 +14,7 @@ MACRO += UPDATE_EO
 MACRO += NDEBUG
 MACRO += CHECK_SPINOR_MATCHING
 MACRO += IO_FLUSH 
+MACRO += DPHI_FLT
```

Comment:
what is this? It's better to not commit changes to this file

Reply:
This is a necessary change. The single precision gauge field u_gauge_f_flt is only allocated in the process_init.c if this compile flag is defined. I had some trouble with this, because for example check_diracoperator_6.c depends on this gauge field being allocated but so far there was no NOCOMPILE to prevent the check from being run without it (I added it now). It is clear, that we don't want to allocate another gauge field if we are not sure that we want to use it. But it is a little intransparent, that you cannot run anything that needs the single precision dirac operator without having added this compile flag. I would keep it there, but maybe disable it. Or by default enable it, so that people don't get cryptic segfaults.

Reply:
ok, you are right, this is not good. we should improve it. This macro shouldn't be necessary at all.
Let's keep it as it is for this PR, but please don't commit the ini file.